actions/hybrid_branch_deploy/action.yml

```diff
@@ -14,13 +14,20 @@ inputs:
 runs:
   using: "composite"
   steps:
-    - name: Checkout
+    - name: Checkout target repo
       uses: actions/checkout@v3
       with:
         ref: ${{ github.head_ref }}
 
+    - name: Checkout action repo
+      uses: actions/checkout@v3
+      with:
+        repository: dagster-io/dagster-cloud-action
+        ref: ${{ github.repository != 'dagster-io/dagster-cloud-action' && '0.1.4' || '' }}
```

Comment:
Do we want to explicitly pass e.g. `github.head_ref` or something here instead of the empty ref?